script/metalsmith-webpack.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable */
```

Comment:
Should we worry about linting this since it'll be living in our code base for...the foreseeable future?

Reply:
Yeah, I probably should just update this section.